Tensile/ClientWriter.py

```diff
@@ -551,6 +551,12 @@ def dataInitParams(problemType):
             ('init-alpha', dataInitName(initAlpha)),
             ('init-beta',  dataInitName(initBeta))]
 
+def clientLogLevelName(num):
```

Comment:
Consider Enum class:

```python3
from enum import Enum
class LogLevel(Enum):
  Error = 0
  Terse = 1
  Verbose = 2
  Debug = 3

print(LogLevel.Debug.name)  # prints Debug
print(LogLevel.Debug.value) # prints 3
```

Reply:
Adopted. And also change the dataInitName to the same method for consistency